www/data/docs_sidebar.yml

```diff
@@ -95,8 +95,29 @@ sidebar_links:
       link: "/docs/reference/#package-contents"
     - title: Supervisor Log Keys
       link: "/docs/reference/#sup-log-keys"
-    - title: Infographics
-      link: "/docs/reference/#infographics"
+  - title: Diagrams
+    link: "/docs/diagrams"
+    sub_links:
+    - title: Architecture Overview
+      link: "/docs/diagrams/#habitat-architecture-overview"
+    - title: Initial Package Build Flow
+      link: "/docs/diagrams/#habitat-initial-package-build-flow"
+    - title: Dependency Update Flow
+      link: "/docs/diagrams/#habitat-dependency-update-flow"
+    - title: Application Rebuild Flow
+      link: "/docs/diagrams/#habitat-application-rebuild-flow"
+    - title: Initial Docker Container Publishing Flow
+      link: "/docs/diagrams/#habitat-initial-docker-container-publishing-flow"
+    - title: Automated Docker Container Publishing Flow
+      link: "/docs/diagrams/#rhabitat-automated-docker-container-publishing-flow"
```

Comment:
Little typo here preventing this link from working: `#rhabitat -> `#habitat`

Reply:
good catch!